deploy/wip-catalog/README.md

```diff
@@ -0,0 +1,43 @@
+# Service Catalog WIP Helm Chart
+
+The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
+
+## Prerequisites
+
+  - [Helm](https://github.com/kubernetes/helm) must be installed in the cluster.
+  - Images must be built from source and pushed to a registry accessible to
+    the cluster (if deploying to a non-local cluster). Example (run from this
+    directory):
+
+    ```
+    $ export REGISTRY=hub.docker.com/<username>
+    $ make -C ../.. push
+    ```
+
+    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
+    to use a _local_ Docker registry.
+
+## Usage
+
+Supported template parameters (values):
+
+### What to deploy, and from where
+
+- `registry`  (optional): Container registry with Service Catalog images; may
+  be left blank if deploying on a local cluster
+- `version`   (optional): Version of Service Catalog (container images) to deploy
+- `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment
+
+### Configuration of the catalog
+
+- `storageType` (optional): The type of storage for the API server to use
+- `verbosity` (optional): The verbosity of logs
```

Comment:
Accepted value? Numeric? Or the familiar "info", "debug", "warn", etc.? Upper or lowercase? Or does it not matter?